android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.kt

```diff
@@ -163,7 +163,7 @@ class ScreenStackHeaderConfig(context: Context) : ViewGroup(context) {
             screenFragment?.setToolbar(toolbar)
         }
 
-        if (isTopInsetEnabled) {
+        if (isTopInsetEnabled && false) {
```

Comment:
This is modified just for testing, will remove it just before merging.

Reply:
Reverted in: https://github.com/software-mansion/react-native-screens/pull/2169/commits/4f34895fbdf72ffad1ede6858b55c8340d664b8b